src/App.css

```diff
@@ -89,6 +112,64 @@
   height: 200px;
 }
 
+.waveform-section {
+  margin: auto;
+}
+.waveform-media > video {
+  width: 100%;
+  padding-right: 0;
+}
+
+.waveform-toolbar {
+  width: 100%;
+  margin: 1rem 0;
+  padding: 0;
+}
+
+/* When the width is less than 768px - small devices */
+@media screen and (max-width: 768px) {
+  #overview-container {
+    height: 45px;
+  }
+
+  #zoomview-container {
+    height: 85px;
+  }
+}
+
+/* When the width is between 768px and 992px - medium device */
+@media screen and (max-width: 991px) and (min-width: 769px) {
+  #overview-container {
+    height: 55px;
+  }
+
+  #zoomview-container {
+    height: 100px;
+  }
+}
+
+/* When the width is between 992px and 1400px - large devices */
+@media screen and (max-width: 1400px) and (min-width: 992px) {
+  #overview-container {
+    height: 60px;
+  }
+
+  #zoomview-container {
+    height: 160px;
+  }
+}
+
+@media screen and (min-width: 1600px) {
```

Comment:
The previous rule has `max-width: 1400 px` so should this be `min-width: 1401px` or is it fine for them to not be contiguous?

Reply:
It's fine for them to be not contiguous. But I think it's safe for us to make it contiguous so that it doesn't behave in an unexpected way that it could impact the usability of the component. I will make this change :+1: